pennylane/templates/embeddings/displacement.py

```diff
@@ -77,11 +77,17 @@ def DisplacementEmbedding(features, wires, method="amplitude", c=0.1):
 
     #############
 
-    constants = c * qml.tape.interfaces.functions.WrapperFunctions.ones_like(features)
+    if qml.tape_mode_active():
+        constants = c * qml.tape.UnifiedTensor(features).ones_like()
+    else:
+        constants = [c] * len(features)
```

Comment:
Actually, now I see it I am quite happy with the forking...easier to delete later.

Reply:
Yeah, I toyed with making a `VariableTensor` subclass as well, but that would be way too much work 😆